public/style.css

```diff
@@ -1,3 +1,29 @@
-body {
```

Comment:
I did the same for now, just created a universal style sheet for dev purposes. I plan to add styled components over the next week or so.